app/controllers/application_controller.rb

```diff
@@ -137,7 +137,9 @@ def request_referrer_is_a_valid_after_login_path?
     end
 
     def inertia_props(props = {})
-      RenderingExtension.custom_context(view_context).merge(props)
+      result = RenderingExtension.custom_context(view_context).merge(props)
+      result[:title] = @title if @title.present?
+      result
```

Comment:
he @title variable was not being passed to the JavaScript frontend at all. It lived only in the Ruby controller - so passed it like this so we don't have to do this in inertia_props in every controller